components/scroll-to-top-button.js

```diff
@@ -1,31 +1,32 @@
 import { useEffect, useState } from "react";
 
 export default function ScrollToButtonButton() {
+  const [hasScrollToTopButton, setHasScrollToTopButton] = useState(false);
 
-    const [hasScrollToTopButton, setHasScrollToTopButton] = useState(false);
+  function toggleScrollTopButton() {
+    setHasScrollToTopButton(
+      document.body.scrollTop > 50 || document.documentElement.scrollTop > 50,
+    );
+  }
 
-    function toggleScrollTopButton() {
-        setHasScrollToTopButton(
-            document.body.scrollTop > 50 || document.documentElement.scrollTop > 50
-        )
-    }
+  useEffect(() => {
+    window.addEventListener("scroll", toggleScrollTopButton);
 
-    useEffect(() => {
-        window.addEventListener('scroll', toggleScrollTopButton);
+    return () => {
+      window.removeEventListener("scroll", toggleScrollTopButton);
+    };
+  }, []);
 
-        return () => {
-            window.removeEventListener('scroll', toggleScrollTopButton);
-
-        }
-    }, [])
-
-    return (
-        <>
-            {hasScrollToTopButton && (
-                <a href="#" className="scroll-top btn-hover">
-                    <i className="lni lni-chevron-up"></i>
-                </a>
-            )}
-        </>
-    )
-}
\ No newline at end of file
+  return (
+    <>
```

Comment:
`<>` should not be needed as you are returning a single root node or `null`

Reply:
I get an error from eslint when I remove the `<>` https://share.zight.com/wbuge4m1

Reply:
you are correct. thanks for checking.